KachnaOnline.Business/Facades/BoardGameFacade.cs

```diff
@@ -0,0 +1,271 @@
+            if (!user.IsInRole(RoleConstants.BoardGamesManager))
+            {
+                throw new NotABoardGamesManagerException();
+            }
+
+            var createdGame = await _boardGameService.CreateBoardGame(_mapper.Map<BoardGame>(game));
+            return _mapper.Map<BoardGameDto>(createdGame);
+        }
+
+        /// <summary>
+        /// Updates a board game with the given ID.
+        /// </summary>
+        /// <param name="user">User requesting the board game update.</param>
+        /// <param name="id">ID of the board game to update.</param>
+        /// <param name="game"><see cref="BoardGameDto"/> representing the new state.</param>
+        /// <exception cref="NotABoardGamesManagerException">When the <paramref name="user"/> is not allowed to update
+        /// the board game (i.e. is not a board game manager).</exception>
+        /// <exception cref="BoardGameNotFoundException">When a board game with the given <paramref name="id"/> does not
+        /// exist.</exception>
+        /// <exception cref="BoardGameManipulationFailedException">When the board game cannot be updated.</exception>
+        /// <exception cref="CategoryNotFoundException">When a category with the ID assigned to the game
+        /// does not exist.</exception>
+        /// <exception cref="UserNotFoundException">When a user with the ID assigned to the game does
+        /// not exist.</exception>
+        public async Task UpdateBoardGame(ClaimsPrincipal user, int id, BoardGameDto game)
+        {
+            if (!user.IsInRole(RoleConstants.BoardGamesManager))
+            {
+                throw new NotABoardGamesManagerException();
+            }
+
+            await _boardGameService.UpdateBoardGame(id, _mapper.Map<BoardGame>(game));
+        }
+
+        /// <summary>
+        /// Updates stock of a board game with the given ID.
+        /// </summary>
+        /// <param name="user">User requesting the board game stock update.</param>
+        /// <param name="id">ID of the board game to update.</param>
+        /// <param name="stock"><see cref="BoardGameStockDto"/> representing the new stock state.</param>
+        /// <exception cref="NotABoardGamesManagerException">When the <paramref name="user"/> is not allowed to update
+        /// the board game stock (i.e. is not a board game manager).</exception>
+        /// <exception cref="BoardGameNotFoundException">When a board game with the given <paramref name="id"/> does not
+        /// exist.</exception>
+        /// <exception cref="BoardGameManipulationFailedException">When the board game cannot be updated.</exception>
+        /// <exception cref="ArgumentException">When any of the <see cref="BoardGameStockDto"/> values are not
+        /// specified.</exception>
+        public async Task UpdateBoardGameStock(ClaimsPrincipal user, int id, BoardGameStockDto stock)
+        {
+            if (stock.InStock is null || stock.Unavailable is null || stock.Visible is null)
```

Comment:
In cases like this, use [pattern matching](https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/functional/pattern-matching): `if (stock is {InStock: null} or {Unavailable: null} or {Visible: null}`)

However, as pointed out in other comments, I don't think this is even neccessary here (a DTO with null values wouldn't get through validation and why are the values even nullable in this DTO in the first place?)